src/actions/channel_categories.ts

```diff
@@ -52,13 +52,45 @@ export function setCategorySorting(categoryId: string, sorting: CategorySorting)
         const state = getState();
         const category = getCategory(state, categoryId);
 
-        return dispatch(updateCategory({
+        dispatch(updateCategory({
```

Comment:
Don't we want most actions to return data? This might just be a TS headache but when we have actions that don't return some `{data: any; error: Error}` the redux connector complains from a TS POV because the return type doesn't match `ActionFunc`.

Reply:
Good catch. I think I removed this by mistake.

In general, I'd like to get away from requiring every action to return something, but that's something for later.

Reply:
Agreed, I'd like to fix the TS headache since there's no requirement to return anything.